dashboard/data/bmg_plate.py

```diff
@@ -148,50 +148,53 @@ def well_to_ids(well_name: str) -> tuple[int, int]:
     tail = well_name[len(head) :]
     return ord(head) - 65, int(tail) - 1
 
-
-def parse_bmg_file(filepath: str) -> tuple[str, np.ndarray]:
+def read_io_file(file: str) -> list[str]:
+        readed = []
+        line = file.readline()
+        while(line):
+            readed.append(line)
+            line = file.readline()
+        return readed
+
+def parse_bmg_file_iostring(filename: str, filecontent: str) -> np.array:
```

Comment:
I'd keep it as `parse_bmg_file` , but it's just a suggestion

Reply:
this name already is used and both functions are slightly different

Reply:
`filecontent` has wrong annotation (it is not a string)
return type should be `np.ndarray`

Reply:
+ missing params description